### DISCUSSION TITLE
Github Discussions - UI issue

### DISCUSSION THREAD

**Discussion by ss-o** [community/community#23505] [New to GitHub]
<p>Hi,</p>
<p>I have got a broken UI after deleting a comment, which seems to persist on all browsers.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/3/732c4452d8a3bb025401234744a2dabdfeab7b37.png" data-download-href="/uploads/short-url/gqRKwP11ri5DG0idtdjta7fRPcr.png?dl=1" title="Screenshot 2022-05-15 at 00-46-05 z-shell_zi 𝕋ℍ𝔼 𝕊𝕎𝕀𝕊𝕊 𝔸ℝ𝕄𝕐 𝕂ℕ𝕀𝔽𝔼 𝔽𝕆ℝ ℤ𝕊ℍ 𝕌ℕ𝕀𝕏 𝕊ℍ𝔼𝕃𝕃" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59910950/181077456-eb5ad0f3-e62b-46dd-939f-c1a0f24febaa.png" alt="Screenshot 2022-05-15 at 00-46-05 z-shell_zi 𝕋ℍ𝔼 𝕊𝕎𝕀𝕊𝕊 𝔸ℝ𝕄𝕐 𝕂ℕ𝕀𝔽𝔼 𝔽𝕆ℝ ℤ𝕊ℍ 𝕌ℕ𝕀𝕏 𝕊ℍ𝔼𝕃𝕃" data-base62-sha1="gqRKwP11ri5DG0idtdjta7fRPcr" width="690" height="299" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/3/732c4452d8a3bb025401234744a2dabdfeab7b37_2_690x299.png, https://aws1.discourse-cdn.com/github/original/3X/7/3/732c4452d8a3bb025401234744a2dabdfeab7b37.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/3/732c4452d8a3bb025401234744a2dabdfeab7b37.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/3/732c4452d8a3bb025401234744a2dabdfeab7b37_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2022-05-15 at 00-46-05 z-shell_zi 𝕋ℍ𝔼 𝕊𝕎𝕀𝕊𝕊 𝔸ℝ𝕄𝕐 𝕂ℕ𝕀𝔽𝔼 𝔽𝕆ℝ ℤ𝕊ℍ 𝕌ℕ𝕀𝕏 𝕊ℍ𝔼𝕃𝕃</span><span class="informations">923×401 37.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can someone advise on this?</p>
<p>Thanks,</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/ss-o">@ss-o</a>,</p>
<p>Thanks for flagging this :bowing_woman:t3: The space to discuss Discussions and any errors now has a new home. I recommend posting your issue  <a href="https://github.com/github/feedback/discussions">here</a> as you may receive better assistance :slight_smile:</p>
<p>Best of luck!</p>

**Comment by ss-o** [comment]
<p>Hey,</p>
<p>Thanks for your respose. :slight_smile:</p>
